### PR TITLE
Set up the stable  Docker repository

### PR DESCRIPTION
**Pull request by nomulex**
Add command to set up the stable  Docker repository

### CODE REVIEW

**Review by newtewt — CHANGES_REQUESTED** [review]
One question inline.

File: self-hosting/prepare-system/ubuntu/prepare.sh [hunk 1]
```diff
@@ -9,6 +9,9 @@ sudo mkdir -p /var/log/medic
 sudo apt-get update
 
 # Install Docker CE
+sudo echo \
```
Comment (newtewt) [inline self-hosting/prepare-system/ubuntu/prepare.sh:22]: I see this as the third step when configuring the repository. Any reason to not include the curl to get the signing?

`curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg`

Comment (newtewt) [reply self-hosting/prepare-system/ubuntu/prepare.sh:22]: Third step from here https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository

Comment (nomulex) [reply self-hosting/prepare-system/ubuntu/prepare.sh:22]: Good call. I have added more pre install steps. Kindly check and let me know.

**Review by newtewt — APPROVED** [review]
👍

**Pull request merged**